src/main/java/hudson/plugins/git/GitPublisher.java

```diff
@@ -573,15 +573,15 @@ public String getDisplayName() {
         private String noteNamespace;
         private boolean noteReplace;
 
-        public String getnoteMsg() {
+        public String getNoteMsg() {
```

Comment:
Right, I understood the motivation, but it still may have broken plugins depending on the old name. Adding fixed getters makes sense in addition to keeping the old ones. They should probably be marked `@deprecated` as their names don't follow convention, and their Javadoc should reference the new alternatives.